File: CompSci 1/Unit 14, Iteration - while/ExamScores/ExamScores.java
import java.util.Scanner;

public class ExamScores {

    private int score = 0;
    private int a = 0;
    private int b = 0;
    private int c = 0;
    private int d = 0;
    private int f = 0;
    private Scanner sc = new Scanner(System.in);

    public void inputAndTally() {
        int grade = 0;
        while (grade != -1) {
            grade = sc.nextInt();
            if (grade >= 90) {
                a++;
            } else if (grade >= 80) {
                b++;
            } else if (grade >= 70) {
                c++;
            } else if (grade >= 60) {
                d++;
            } else if (grade >= 0){
                f++;
            }
        }
    }

    public int getTotalScores()
    {
        return (a + b + c + d + f);
    }

    public void printSummary()
    {
        System.out.println("============================");
        System.out.println("       Summary Report");
        System.out.println("============================");
        System.out.println("Total number of grades = " + getTotalScores());
        System.out.println("Number of As = " + a);
        System.out.println("Number of Bs = " + b);
        System.out.println("Number of Cs = " + c);
        System.out.println("Number of Ds = " + d);
        System.out.println("Number of Fs = " + f);
    }

    public static void main(String[] args) {
        ExamScores app = new ExamScores();
        app.inputAndTally();
        app.printSummary();
    }
}

File: CompSci 1/Unit 14, Iteration - while/ExamScores/ExamScores.kt
import java.util.Scanner


private val score = 0
private var a = 0
private var b = 0
private var c = 0
private var d = 0
private var f = 0
private val sc = Scanner(System.`in`)

val totalScores: Int
    get() = a + b + c + d + f

fun inputAndTally() {
    var grade = 0
    while (grade != -1) {
        grade = sc.nextInt()
        if (grade >= 90) {
            a++
        } else if (grade >= 80) {
            b++
        } else if (grade >= 70) {
            c++
        } else if (grade >= 60) {
            d++
        } else if (grade >= 0){
            f++
        }
    }
}

fun printSummary() {
    println("============================")
    println("       Summary Report")
    println("============================")
    println("Total number of grades = $totalScores")
    println("Number of As = $a")
    println("Number of Bs = $b")
    println("Number of Cs = $c")
    println("Number of Ds = $d")
    println("Number of Fs = $f")
}

fun main(args: Array<String>) {
    inputAndTally()
    printSummary()
}